lib/galaxy/tool_util/linters/outputs.py

```diff
@@ -105,10 +116,12 @@ def __check_format(node, lint_ctx, allow_ext=False):
     if fmt is None:
         fmt = node.attrib.get("format")
     if fmt == "input":
-        lint_ctx.error(
-            f"Using format='input' on {node.tag}, format_source attribute is less ambiguous and should be used instead.",
-            node=node,
-        )
+        message = f"Using format='input' on {node.tag} is deprecated. Use the format_source attribute."
+        if profile <= "16.01":
```

Comment:
I think you want to cast these to floats before doing the comparison.

Reply:
Good point. I just checked. One finds comparison with float and `Version(str(profile)) <= Version("16.01")` in the sources. Which is better?

One also finds comparison with string .. I can fix this as a follow up.

Reply:
I guess `Version` is better.